src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/column/main.js

```diff
@@ -45,18 +47,31 @@ define([
 
         ACTION_ICON_VIEW = 'fa-eye',
 
+        ACTION_ICON_INFO = 'fa-info',
+
+
+        brokenPageDialogTranslations = {
```

Comment:
Wouldn't it be a better idea to just name this `translations`? Then it would be consistent with the other components. And I am wondering why you are not using the hook you've developed some time ago.